pymc3/distributions/continuous.py

```diff
@@ -203,26 +201,22 @@ class Normal(Continuous):
         Precision (tau > 0).
     """
 
+    rng = stats.norm.rvs
```

Comment:
I pass arguments to self.parent ordered dict and then don't care about their names user should not too.